ci/ray_ci/move_test_state.py

```diff
@@ -0,0 +1,4 @@
+from ci.ray_ci.automation.state_machine_bot import main
```

Comment:
just remove this wrapper. it is a script (with no test). you do not really need this wrapper.